epochStart/bootstrap/syncEpochStartMeta_test.go

```diff
@@ -0,0 +1,115 @@
+package bootstrap
+
+import (
+	"errors"
+	"testing"
+	"time"
+
+	"github.com/ElrondNetwork/elrond-go/core/check"
+	"github.com/ElrondNetwork/elrond-go/data/block"
+	"github.com/ElrondNetwork/elrond-go/epochStart/mock"
+	"github.com/ElrondNetwork/elrond-go/p2p"
+	"github.com/ElrondNetwork/elrond-go/process/economics"
+	"github.com/stretchr/testify/require"
+)
+
+func TestNewEpochStartMetaSyncer_ShouldWork(t *testing.T) {
+	t.Parallel()
+
+	args := getEpochStartSyncerArgs()
+	ess, err := NewEpochStartMetaSyncer(args)
+	require.NoError(t, err)
+	require.False(t, check.IfNil(ess))
+}
+
+func TestEpochStartMetaSyncer_SyncEpochStartMeta_ErrRegisterMessengerProcessorShouldErr(t *testing.T) {
+	t.Parallel()
+
+	expectedErr := errors.New("expected error")
+
+	args := getEpochStartSyncerArgs()
+	messenger := &mock.MessengerStub{
+		RegisterMessageProcessorCalled: func(_ string, _ p2p.MessageProcessor) error {
+			return expectedErr
+		},
+	}
+	args.Messenger = messenger
+	ess, _ := NewEpochStartMetaSyncer(args)
+
+	mb, err := ess.SyncEpochStartMeta(1 * time.Second)
```

Comment:
use time.Second instead of 1 * time.Second

Reply:
done